examples/evaluation/trpcagent/servers/langgraph/protocol.py

```
@@ -0,0 +1,147 @@
+                {"SurfaceID": MODEL_SURFACE_ID, "NodeID": APP_NAME, "Type": "model", "Value": {"Model": {"Name": MODEL_NAME}}},
+            ],
+        }
+    }
+
+
+def run_response(request: dict) -> tuple[int, dict]:
+    request_id = request_id_from(request)
+    invocation_id = request_id
+    try:
+        output, usage = run_agent(request)
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)